sdk/java/runtime/template/pom.xml

```
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>io.dagger.sample</groupId>
```

Maybe
```suggestion
    <groupId>io.dagger.module</groupId>
```

For example, for `dagger init --sdk=java --name=my-module`:
```xml
<groupId>io.dagger.module</groupId>
   <artifactId>my-module</artifactId>
    <version>1.0-SNAPSHOT</version>
    <name>my-module</name>
```
If that conflicts with SDK plugins or something, then maybe something else. I don't see users coming and editing out the "sample" here. Most people just change the generated functions, but mostly leave the rest from the template.

---

I _think_ that's quite common on java world to have people using their own namespace. Like `com.company.project`. And I expected people to change it. But I can find something better than `sample`. Maybe I'll avoid completely `io.dagger` to not have conflicts at all 🤔

---

In other SDKs, when there wasn't support for custom names, we conventionalized on "main". So there's lots of modules out there where the language's package name is "main", even when the dagger module's name is something else.

In this case, whatever is more familiar to Java users takes priority. But I suspect lots of people won't be changing that value. Or does it become super visible in the IDE?

---

This is now `io.dagger.modules.usermodule` with `usermodule` based on the user input (the name of the folder, lowercase, no `-`, no `_`. That way it fits the java convention about package naming and doesn't conflict with `io.dagger.module` package.